packages/Webkul/User/src/Repositories/AdminRepository.php

```diff
@@ -2,7 +2,11 @@
 
 namespace Webkul\User\Repositories;
 
+use Illuminate\Support\Facades\Mail;
+use PragmaRX\Google2FA\Google2FA;
```

Comment:
Move all the logic for google2fa class from repository to TwoFactorAuthentication facade in user pacakge.